hc/front/views.py

```diff
@@ -32,6 +33,8 @@ def my_checks(request):
     q = Check.objects.filter(user=request.team.user).order_by("created")
     checks = list(q)
 
+    allowed_checks = MemberAllowedChecks.objects.filter(user=request.user).values_list('check_id_id', flat=True)
```

Comment:
We should update `check_id_id` when we update `check_id` to `check` in `MemberAllowedChecks ` Model